### PR TITLE
Issue #29 Adjust :bootstrappers params on api containers

### PR DESCRIPTION
**Pull request by fadlinurhasan**

### CODE REVIEW

File: app/controllers/api/v2/ext_app/containers_controller.rb [hunk 1]
```diff
@@ -55,11 +55,7 @@ def container_params
           :fingerprint,
           :alias
         ]},
-        {bootstrappers: [
-          :bootstrap_type,
-          :bootstrap_cookbooks_url,
-          :bootstrap_attributes
-        ]}
+        :bootstrappers
```
Comment (giosakti) [inline app/controllers/api/v2/ext_app/containers_controller.rb]: why don't we make this explicit?

File: app/models/container.rb [hunk 2]
```diff
@@ -57,6 +57,8 @@ class Container < ApplicationRecord
   # Setup additional methods
   #
   def self.create_with_source(cluster_id, params)
+    param_bootstrappers = params.dig(:bootstrappers)
+    bootstrappers = JSON.parse(param_bootstrappers) if param_bootstrappers.present?
```
Comment (giosakti) [inline app/models/container.rb]: why do we need to parse? the type is json right?

File: app/controllers/api/v2/ext_app/containers_controller.rb (hunk 1 above)
Comment (fadlinurhasan) [reply app/controllers/api/v2/ext_app/containers_controller.rb]: Hi mas, We readjust this change to https://github.com/pathfinder-cm/pathfinder-mono/pull/30/commits/37d014568fb4e923f66b72e67a77ba0263904b3e. 

Can you please check.

File: app/models/container.rb (hunk 2 above)
Comment (fadlinurhasan) [reply app/models/container.rb]: We undo this changes related to this commit https://github.com/pathfinder-cm/pathfinder-mono/pull/30/commits/37d014568fb4e923f66b72e67a77ba0263904b3e
Can you please check?

**Pull request merged**